website/docs/r/cci_network_v1.html.md

```diff
@@ -1,37 +1,39 @@
 ---
 layout: "huaweicloud"
-page_title: "Huaweicloud: huaweicloud_cci_network_v1"
-sidebar_current: "docs-huaweicloud-resource-cci-network-v1"
+page_title: "Huaweicloud: huaweicloud_cci_network"
+sidebar_current: "docs-huaweicloud-resource-cci-network"
 description: |-
   Provides Cloud Container Instance(CCI) resource.
 ---
 
-# huaweicloud_cci_network_v1
+# huaweicloud_cci_network
```

Comment:
the resource should named as `huaweicloud_cci_network_v1`, please keep it the same.